presto-parser/src/main/java/com/facebook/presto/sql/ExpressionFormatter.java

```diff
@@ -566,6 +570,43 @@ static String formatSortItems(List<SortItem> sortItems, boolean unmangleNames)
                 .iterator());
     }
 
+    static String formatGroupBy(List<GroupingElement> groupingElements)
+    {
+        checkState(groupingElements.stream()
+                .filter(groupingElement -> !(groupingElement instanceof SimpleGroupBy))
+                .count() <= 1, "multiple groupings are not yet supported");
+
+        List<Expression> simpleGroupings = groupingElements.stream()
+                .filter(groupingElement -> groupingElement instanceof SimpleGroupBy)
```

Comment:
Because of the change you proposed to the grammar, a `GROUP BY a, b` contains two grouping elements - a SimpleGroupBy of a, and a SimpleGroupBy of b.